client/modules/IDE/selectors/project.js

```diff
@@ -2,6 +2,7 @@ import { createSelector } from 'reselect';
 
 export const selectProjectOwner = (state) => state.project.owner;
 export const selectProjectId = (state) => state.project.id;
+export const selectProjectName = (state) => state.project.name;
```

Comment:
❤️

Reply:
hehe, at that time I didn't knew how to cherry pick